wger/manager/models.py

```diff
@@ -718,6 +718,11 @@ class WorkoutLog(models.Model):
     '''
 
     date = Html5DateField(verbose_name=_('Date'))
+    '''
+    This is the session the log belongs to
+    '''
+    session = models.ForeignKey('WorkoutSession',
```

Comment:
Excellent Work.  
Looking at line number 724 of `wger/manager/models.py`
I thought the foreign key model should be the class itself not a string of the name of the class
`models.ForeignKey('WorkoutSession',` *=>* `models.ForeignKey(WorkoutSession,`

Reply:
Other than what @martinMutuma  has pointed out, I think this is good to go.
Run the tests after the change so as to see if things break

Reply:
Referencing the foreign key model as the class itself results in an undefined name situation and it also can't be imported in the same models.py where it has been defined as this will result in a cyclic error. The solution to this is to call it as a string.